test/db2.js

```diff
@@ -63,6 +63,10 @@ tape('db2 friends test', function (t) {
   let sbot = Server({
     keys: alice,
     db2: true,
+    friends: {
+      hookAuth: false,
+      hookReplicate: false
+    },
```

Comment:
These changes are required because the tests were relying on the glue code **not** running (because we don't have ssb-replicate installed, which the glues need). So here we have to opt-out.